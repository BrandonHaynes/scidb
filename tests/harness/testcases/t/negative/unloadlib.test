# ticket no. 1403

--setup
--start-query-logging

--test
--error --code=scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE "unload_library('')"

--error --code=scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE "unload_library('librational')"

--error --code=scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE "unload_library('abcxyz')"

--cleanup
--stop-query-logging

