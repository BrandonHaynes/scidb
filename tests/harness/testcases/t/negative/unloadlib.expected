SCIDB QUERY : <unload_library('')>
[An error expected at this place for the query "unload_library('')". And it failed with error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE. Expected error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE.]

SCIDB QUERY : <unload_library('librational')>
[An error expected at this place for the query "unload_library('librational')". And it failed with error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE. Expected error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE.]

SCIDB QUERY : <unload_library('abcxyz')>
[An error expected at this place for the query "unload_library('abcxyz')". And it failed with error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE. Expected error code = scidb::SCIDB_SE_PLUGIN_MGR::SCIDB_LE_CANT_UNLOAD_MODULE.]

